src/ToricVarieties/cohomCalg/ShiftedCones/attributes.jl

```diff
@@ -0,0 +1,158 @@
+############################
+# Attributes
+############################
+
+@doc Markdown.doc"""
+    gens(sc::ShiftedCone)
+
+Return the generators of the shifted cone `sc`.
+
+# Examples
+```jldoctest
+julia> ray_generators = [[1,2,3],[4,5,6]]
+2-element Vector{Vector{Int64}}:
+ [1, 2, 3]
+ [4, 5, 6]
+
+julia> shift = [-3,-2,-1]
+3-element Vector{Int64}:
+ -3
+ -2
+ -1
+
+julia> sc = ShiftedCone(ray_generators, shift)
+A shifted cone in ambient dimension 3
+
+julia> gens(sc)
+[1   2   3]
+[4   5   6]
+```
+"""
+function gens(sc::ShiftedCone)
+    return sc.gens::fmpz_mat
```

Comment:
But `sc.gens` is already of type `fmpz_mat`, so the type assertion here should not be needed. Am I missing something?

Reply:
I think you are right. This is not needed. Will update this shortly (and check also in other places of the ToricVarieties).